screens/LoginScreen.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useRef } from "react";
+import {
+  Image,
+  KeyboardAvoidingView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import { generalStyles } from "../shared/styles";
+
+const LoginScreen = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const passwordInput = useRef(null);
+
+  return (
+    <KeyboardAvoidingView behavior="padding" style={generalStyles.center}>
+      <View style={styles.loginContainer}>
+        <TextInput
+          style={styles.input}
+          placeholder="Email"
+          placeholderTextColor="rgba(0,0,0,0.4)"
+          onSubmitEditing={() => passwordInput.current.focus()}
+          keyboardType="email-address"
+          autoCapitalize="none"
+          autoCompleteType={"email"}
+          blurOnSubmit={true}
+          value={email}
+          onChangeText={email => setEmail(email)}
+        />
+
+        <TextInput
+          style={styles.input}
+          placeholder="Password"
+          secureTextEntry
+          placeholderTextColor="rgba(0,0,0,0.4)"
+          autoCapitalize="none"
+          ref={passwordInput}
+          value={password}
+          onChangeText={password => setPassword(password)}
+        />
+        <TouchableOpacity
```

Comment:
I think we should use TouchableOpacity for iOS and TouchableNativeFeedback for Android, so that we have a ripple effect on Android